src/screens/SingleSpeakerScreen.tsx

```diff
@@ -1,227 +1,268 @@
-                            <TouchableOpacity onPress={() => navigation.goBack()}>
-                                <ArrowLeftIcon  color={colors.DROIDCONKE_PEARL}/>
-                            </TouchableOpacity>
-                            <Text style={styles.speakerheadertext}> Speaker</Text>
-                        </View>
-                        
-                        <Image style={styles.profilepic} source={item.speakerimg}/>
-                    </ImageBackground>
-                </View>
-                <View style={styles.sectiontwo}>
-                    <View style={styles.sectiontwo_subsectionone}>
-                        <View style={styles.sectiontwo_subsectiononeofone}>
-                            <AndroidIcon width={27} height={27} color={colors.DROIDCONKE_BRICK_RED}/>
-                            <Text style={styles.sectiontwo_subsectionone_itemone}>Speaker:</Text>
-                        </View>
-                        <Text style={styles.sectiontwo_subsectionone_itemtwo}>{item.speakername}</Text>
-                        <Text style={styles.sectiontwo_subsectionone_itemthree}>{item.speakeroccupation}</Text>
-                        <Text style={styles.sectiontwo_subsectionone_itemthree}>
-                            {myloop}
-                        </Text>
-                    </View>
-                    <View style={styles.sectiontwo_subsectiontwo}>
-                        <Text style={styles.sectiontwo_subsectiontwo_header}>Bio</Text>
-                        <ScrollView>
-                        <Text style={styles.sectiontwo_subsectiontwo_paragraph}>
-                            {textElements}
-                        </Text>
-                        </ScrollView>
-                        
-                    </View>
-                </View>
-                <View style={styles.sectionthree}>
-                    <Text style={styles.sectionthree_text}>Twitter Handle</Text>
-                    <TouchableOpacity onPress={() => {alert("Twitter handle pressed!")}}>
-                    <View style={styles.sectionthree_subsectionone}>
-                        <TwitterIcon width={30} height={25} color={colors.DROIDCONKE_BLUE}/>
-                        <Text style={{...styles.sectionthree_text, color: colors.DROIDCONKE_BLUE}}>{item.speakerhandle}</Text>
-                    </View>
-                    </TouchableOpacity>
-                </View>
-            </View>
-        </SafeAreaView>
-    )
+						<Image style={styles.profilepic} source={item.speakerimg} />
+					</ImageBackground>
+				</View>
+				<View style={styles.sectiontwo}>
+					<View style={styles.sectiontwo_subsectionone}>
+						<View style={styles.sectiontwo_subsectiononeofone}>
+							<AndroidIcon
+								width={27}
+								height={27}
+								color={colors.DROIDCONKE_BRICK_RED}
+							/>
+							<Text style={styles.sectiontwo_subsectionone_itemone}>
+								Speaker:
+							</Text>
```

Comment:
Good work on killing two birds with 1 stone!
At the moment, this is the titles shown on the team member screen
![photo_2022-10-10_11-05-32](https://user-images.githubusercontent.com/28436092/194822309-6742f469-cc75-42c1-aca0-8c6402ffb465.jpg)

This is what the titles should display based on the design
![image](https://user-images.githubusercontent.com/28436092/194822445-2923c4ed-0896-4e62-ad64-39ff210111a2.png)

Notice is shows `Sponsorship Lead or some dynamic title`

Reply:
Fixed. See screenshots below: 

### Speaker Bio 

![Screenshot 2022-10-10 at 21-47-51 Speaker-BioScreen jpeg (JPEG Image 606 × 1280 pixels) — Scaled (55%)](https://user-images.githubusercontent.com/59362946/194933514-2aa28387-0df0-47c5-9b8a-ed3f4dea14ef.png)



### Team Member Bio 



![Screenshot 2022-10-10 at 21-47-10 Team-BioScreen jpeg (JPEG Image 606 × 1280 pixels) — Scaled (55%)](https://user-images.githubusercontent.com/59362946/194933614-5f9f61ba-37e9-4019-bb4c-54f593e449e4.png)